client/me/developer/features/style.scss

```
@@ -0,0 +1,56 @@
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@automattic/typography/styles/variables";
+
+.developer-features-list {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	grid-gap: 24px;
+	padding-bottom: 24px;
+	font-family: "SF Pro Text", $sans;
+
+	.developer-features-list__item {
+		display: flex;
+		flex-direction: column;
+		border-radius: 2px;
+		border: 1px solid #dcdcde;
+		background: #fff;
+		padding: 23px 23px 0 23px;
+
+		.developer-features-list__item-title {
+			padding-bottom: 8px;
+			font-weight: 500;
+			font-size: rem(16px);
+			letter-spacing: -0.32px;
+			line-height: 24px;
+		}
+		.developer-features-list__item-description {
+			padding-bottom: 8px;
+			font-size: rem(14px);
+			color: #757575;
+			line-height: 20px;
+			letter-spacing: -0.15px;
+		}
+		.developer-features-list__item-learn-more {
+			padding-bottom: 23px;
+			margin-top: auto;
+
+			a {
+				line-height: 13px;
+				font-size: rem(13px);
+				color: #c9356e;
```

This will look broken on some colour schemes.

```suggestion
				color: var(--color-accent);
```